README.md

```diff
@@ -1,2 +1,3 @@
 # hello-world
 hellloo world! 
+What's thisss?
```

Comment:
I can comment on your change and request changing if you assign me as a reviewer, like this.

Reply:
You can mark it as resolve. You can configure the repo so that unreviewed pull requests or pull requests with unresolved comments cannot be merged.